partiql-parser/src/main/kotlin/org/partiql/parser/internal/PartiQLParserDefault.kt

```diff
@@ -574,7 +574,7 @@ internal class PartiQLParserDefault : PartiQLParser {
             }
         }
 
-        override fun visitQualifiedName(ctx: org.partiql.parser.antlr.PartiQLParser.QualifiedNameContext) = translate(ctx) {
+        override fun visitQualifiedName(ctx: org.partiql.parser.internal.antlr.PartiQLParser.QualifiedNameContext) = translate(ctx) {
```

Comment:
```suggestion
        override fun visitQualifiedName(ctx: GeneratedParser.QualifiedNameContext) = translate(ctx) {
```

For this and other place applicable, could change to `GeneratedParser`.